TypeCobol.Test/Utils/Comparator.cs

```diff
@@ -211,7 +215,8 @@ public int GetTestCount()
 				    }
 				}
 			}
-			if (errors.Length > 0) throw new Exception(errors.ToString());
+			if (errors.Length > 0)
+                throw new Exception(errors.ToString());
```

Comment:
why the indent change ?

Reply:
I needed to set a breakpoint here.

Reply:
VS allows to set breakpoint on parts of line, anyway it does not matter.